android/src/main/java/com/twilio/twilio_voice/AnswerJavaActivity.java

```diff
@@ -196,12 +196,10 @@ private void acceptCall() {
         if (TwilioVoicePlugin.hasStarted) {
             finish();
         } else {
-            Log.d(TAG, "Answering call");
-
-            activeCallInvite.accept(this, callListener);
```

Comment:
I think this is the only big change, All other changes look good. I don't have an Android phone right now to test this, so I will tomorrow

Reply:
@diegogarciar I'll test a bit later - focusing on iOS atm but will review Android later as a second opinion

Reply:
Hi, just tested it, and it logs that both the AnswerActivity and TwilioPlugin answer the call at the same time, this was the purpose for the if else. if the plugin hasn't started (app was killed), the `AnswerJavaActivity` answers, otherwise the plugin does. Is there a specific reason for this change?

<img width="907" alt="Screen Shot 2022-02-01 at 8 20 24 p m" src="https://user-images.githubusercontent.com/17276155/152088194-21edda6f-d019-4d44-b4a9-055b23d8ce22.png">